index.html

```diff
@@ -1,657 +1,1295 @@
+                      <div id="title-36fabd18c5" class="cmp-title">
+                        <h3 class="cmp-title__text">Join Thousands of Leaders Leaving a Lasting Impact</h3>
+                      </div>
                     </div>
-                    <div class="btn-wrap">
-                        <div class="button">
-                            <a href="" id="" class="btn-link" data-cmp-clickable="" data-cmp-data-layer="">
-                                <span class="btn-text">Subscribe to Our Newsletter</span>
+
+                    <div class="responsivegrid aem-GridColumn aem-GridColumn--default--12 layout-grid">
+                      <div id="container-32aff43d65" class="cmp-container">
+                        <div class="aem-Grid aem-Grid--12 aem-Grid--default--12 button-wrap">
+                          <div class="button btn-link aem-GridColumn aem-GridColumn--default--12 layout-grid">
+                            <a id="button-1cbc85b1e4" class="cmp-button" href="/content/leaderimpact/lang/en2/stories.html">
+                              <span class="cmp-button__text">Subscribe to Our Newsletter</span>
                             </a>
-                        </div>
-                        <div class="button">
-                            <a href="" id="" class="btn-link" data-cmp-clickable="" data-cmp-data-layer="">
-                                <span class="btn-text">Join a Group</span>
+                          </div>
+
+                          <div class="button btn-link aem-GridColumn aem-GridColumn--default--12 layout-grid">
+                            <a id="button-61928ed28f" class="cmp-button" href="/content/leaderimpact/lang/en2/join-a-group.html">
+                              <span class="cmp-button__text">Join a Group</span>
                             </a>
-                        </div>
-                        <div class="button">
-                            <a href="" id="" class="btn-link" data-cmp-clickable="" data-cmp-data-layer="">
-                                <span class="btn-text">Start a Group</span>
+                          </div>
+
+                          <div class="button btn-link aem-GridColumn aem-GridColumn--default--12 layout-grid">
+                            <a id="button-9752993433" class="cmp-button" href="/content/leaderimpact/lang/en2/start-a-group.html">
+                              <span class="cmp-button__text">Start a Group</span>
                             </a>
+                          </div>
+
+                          <div class="responsivegrid aem-GridColumn aem-GridColumn--default--12 layout-grid navigation-wrap">
+                            <div id="container-d76502f407" class="cmp-container">
+                              <div class="aem-Grid aem-Grid--12 aem-Grid--default--12 nav-content-wrap">
+                                <div class="navigation nav-wrap aem-GridColumn aem-GridColumn--default--12 layout-grid">

```

Comment:
You're missing `<div id="container-93b0189817" class="cmp-container">` between these two.

Reply:
Done